aces/Loris
# [RBData/Candidate] Remove multiple first visits in session

## Brief summary of changes

Follows  up on suggestion made in https://github.com/aces/Loris/pull/4075 to catch error  

```
Attempt to use pselectRow on a query that returns multiple rows
```

thrown in `Database.class.inc`, when calling function Candidate->getFirstVisit()  in `assign_missing_instruments.php`, when multiple first visits are returned, and throw new LorisException with more meaningful message within the function getFirstVisit().

#### Testing instructions (if applicable)

1. On minor branch, try `php assign_missing_instruments.php` and see if you get the error:

```
PHP Fatal error:  Uncaught DomainException: Attempt to use pselectRow on a query that returns multiple rows in /var/www/loris/php/libraries/Database.class.inc:818
```

If you do, switch to this PR's branch, and run `php assign_missing_instruments.php` again. You should instead get something similar to:

```
PHP Fatal error:  Uncaught LorisException: Candidate 300299 has multiple first visits recorded where only 1 is expected. in /var/www/loris/php/libraries/Candidate.class.inc:622
```

2. If you don't have a candidate in your database that has multiple first visits (you don't get the above errors), mock one by duplicating values in 'VisitNo' column in `session` table for a candidate, and repeat step 1.

#### Links to related tickets (GitHub, Redmine, ...)

https://github.com/aces/Loris/pull/4075
